app/form_models/publishers/vacancy_statistics_form.rb

```diff
@@ -0,0 +1,13 @@
+class Publishers::VacancyStatisticsForm
+  include ActiveModel::Model
+
+  attr_accessor :hired_status, :listed_elsewhere
+
+  validate :all_questions_completed
```

Comment:
Can we just validate the presence of each attribute and update the spec to use shoulda-matchers?

Reply:
It would be nice, but the error summary would need to have some way of linking to the fields of the correct form (there being multiple forms on the same page), which they don't currently. That's part of why I chose to keep the error message general (and attach it to `:base`).

Reply:
Opened an issue https://github.com/DFE-Digital/govuk_design_system_formbuilder/issues/252